docs/rules/member-ordering.md

```diff
@@ -0,0 +1,584 @@
+# Enforces a standard member declaration order.
+
+A consistent ordering of fields, methods and constructors can make interfaces, type literals, classes and class 
+expressions easier to read, navigate and edit.
+
+## Rule Details
+
+This rule aims to standardise the way interfaces, type literals, classes and class expressions are structured.
+
+## Options
+
+This rule, in its default state, does not require any argument, in which case the following order is enforced:          
+- `public-static-field`
+- `protected-static-field`
+- `private-static-field`
+- `public-instance-field`
+- `protected-instance-field`
+- `private-instance-field`
+- `public-field` (disregard of the scope)
```

Comment:
I updated the documentation, not really sure if this will really help... 
Do let me know if anyone has a better suggestion...